wire-ios/Tests/TestPlans/AllTests.xctestplan

```diff
@@ -64,6 +64,22 @@
         "identifier" : "E6579E352AFF9B30004E7FD8",
         "name" : "Wire-iOS UnitTests"
       }
+    },
+    {
+      "parallelizable" : true,
+      "target" : {
+        "containerPath" : "container:..\/WireUI",
+        "identifier" : "WireReusableUIComponentsTests",
+        "name" : "WireReusableUIComponentsTests"
+      }
+    },
+    {
+      "parallelizable" : true,
+      "target" : {
+        "containerPath" : "container:..\/WireUI",
+        "identifier" : "WireDesignTests",
+        "name" : "WireDesignTests"
+      }
```

Comment:
so do I understand it correctly that running allTests so cmd+U on Wire-iOS runs WireUI targets too?

Reply:
Yes, unless you prefer not to have this behaviour.